### ISSUE TITLE
latex formatting - make fig 2 and 3 single column width

### ISSUE THREAD

**Issue by sonyahanson** [choderalab/dispensing-errors-manuscript#31]
Maybe this is not relevant for actual JCAMD publication, but would e nice if the 'programmatic x-based dilution series' figures didn't take up a whole page width, but a single column width.

**Comment by sonyahanson** [comment]
Also relevant to fig 4 - dilution effect illustration

**Comment by jchodera** [comment]
I think you just change `figure*` to `figure` in [this block](https://github.com/choderalab/dispensing-errors-manuscript/blob/master/manuscript/dispensing-errors.tex#L269-L278) to get column-width figures:

``` latex
\begin{figure*}[tb]
    \includegraphics[width=0.5\textwidth]{../figures/dilution.pdf}

  \caption{{\bf A programmatic tip-based dilution series.}
  In this dilution series we start with our initial well at concentration, $C0$, with initial volume, $Vinitial$. A volume, $Vtransfer$, is then transferred into a volume, $Vbuffer$, of buffer. 
  In this case $Vtransfer$ and $Vbuffer$ are equal, so the concentration of the second well will be $C0/2$. This transfer is then repeated for all subsequent wells, the number of which is defined by $ndilutions$. 
  From the last well the volume $Vtransfer$ is removed, so that all wells have the same final volume of $Vtransfer=Vbuffer$.
  }
  \label{fig:dilution}
\end{figure*}
```

You will also want to change the figure width to be a bit less than column width.

**Comment by sonyahanson** [comment]
I think this is done.

**Comment by jchodera** [comment]
Yes!  Fixed in #34
